packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -96,7 +96,7 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
                     name="inviteUser"
                     placeholder="email@example.com"
                     required
-                    onChange={onChange}
+                    onChange={(e) => onChange(e.target.value.trim().toLowerCase())}
```

Comment:
Can add similar logic in rules as well so that the error can be shown without needing validation from server.
<img width="1035" alt="Screenshot 2023-05-10 at 11 23 21 AM" src="https://github.com/calcom/cal.com/assets/1780212/b12f7e39-e5b4-4261-a181-1b15705276bf">